packages/expo-notifications/android/src/androidTest/kotlin/expo/modules/notifications/ExpoNotificationBuilderTest.kt

```diff
@@ -0,0 +1,89 @@
+package expo.modules.notifications
+
+import android.app.Notification
+import android.content.Context
+import android.graphics.Color
+import android.net.Uri
+import androidx.core.app.NotificationCompat
+import androidx.test.filters.SmallTest
+import androidx.test.platform.app.InstrumentationRegistry
+import expo.modules.notifications.notifications.enums.NotificationPriority
+import expo.modules.notifications.notifications.model.NotificationContent
+import expo.modules.notifications.notifications.model.NotificationRequest
+import expo.modules.notifications.notifications.presentation.builders.ExpoNotificationBuilder
+import kotlinx.coroutines.runBlocking
+import org.json.JSONObject
+import org.junit.Assert.assertArrayEquals
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+
+@SmallTest
+class ExpoNotificationBuilderTest {
+
+  private lateinit var context: Context
+  private lateinit var notificationBuilder: ExpoNotificationBuilder
+  private lateinit var notificationContent: NotificationContent
+  private lateinit var notificationRequest: NotificationRequest
+
+  @Before
+  fun setup() {
+    context = InstrumentationRegistry.getInstrumentation().targetContext
+
+    notificationContent = NotificationContent.Builder()
+      .setTitle("Test Title")
+      .setSubtitle("Test Subtitle")
+      .setText("Test Text")
+      .setSticky(true)
+      .setColor(Color.RED)
+      .setAutoDismiss(true)
+      .setSound(Uri.parse("content://media/external/audio/media/123"))
+      .setBody(JSONObject("{\"key\":\"value\"}"))
+      .setPriority(NotificationPriority.HIGH)
+      .setVibrationPattern(longArrayOf(100, 200, 300, 400))
+      .build()
+
+    notificationRequest = NotificationRequest("test-1", notificationContent, null)
+    val exNotification =
+      expo.modules.notifications.notifications.model.Notification(notificationRequest)
+
+    notificationBuilder = ExpoNotificationBuilder(context)
+    notificationBuilder.setNotification(exNotification)
```

Comment:
I'm not sure if this code should be called inside of the `@Before` function. This function would be fired for every test in this class and it seems very specific to one case. Could you replace it with a new function, for instance: `createTestNotificationBuilder(): ExpoNotificationBuilder`, and call it at the start of the test function?